pkg/lightning/backend/kv/session.go

```diff
@@ -312,6 +311,7 @@ func (*sessEvalContext) GetOptionalPropProvider(exprctx.OptionalEvalPropKey) (ex
 	return nil, false
 }
 
+// Deprecated: `session` will be removed soon.
```

Comment:
I have decided to remove the old `session` from the next PR. In this PR, it is used in some tests to check the old and new session's behavior are the same